packages/zowe-explorer/src/shared/actions.ts

```diff
@@ -209,9 +209,7 @@ export async function openRecentMemberPrompt(
                 );
                 return;
             } else if (choice instanceof FilterDescriptor) {
-                if (quickpick.value) {
-                    pattern = quickpick.value;
-                }
+                pattern = quickpick.value;
```

Comment:
Not sure if it matters here but in case value is undefined/null, should we default to `""` ?